docs/release-1.3/README.md

```diff
@@ -0,0 +1,309 @@
+    ```
+    **NOTE:** If you don't want to use the default values, omit the `--use-default` and the script will prompt you for input.
+
+    The contents will vary depending on the configuration in the cluster. The following list details all the keys that can appear in the secret:
+
+    > - `BACKEND_SECRET`: Value is randomly generated at script execution. This is the only mandatory key required to be in the secret for the RHDH Operator to start.
+    > - `K8S_CLUSTER_URL`: The URL of the Kubernetes cluster is obtained dynamically using `oc whoami --show-server`.
+    > - `K8S_CLUSTER_TOKEN`: The value is obtained dynamically based on the provided namespace and service account.
+    > - `GITHUB_TOKEN`: This value is prompted from the user during script execution and is not predefined.
+    > - `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET`: The value for both these fields are used to authenticate against GitHub. For more information open this [link](https://backstage.io/docs/auth/github/provider/).
+    > - `ARGOCD_URL`: This value is dynamically obtained based on the first ArgoCD instance available.
+    > - `ARGOCD_USERNAME`: Default value is set to `admin`.
+    > - `ARGOCD_PASSWORD`: This value is dynamically obtained based on the first ArgoCD instance available.
+
+    Keys will not be added to the secret if they have no values associated. So for instance, when deploying in a cluster without the GitOps operators, the `ARGOCD_URL`, `ARGOCD_USERNAME` and `ARGOCD_PASSWORD` keys will be omited in the secret.
+
+    Sample of a secret created in a GitOps environment:
+
+    ```console
+    $> oc get secret -n rhdh-operator -o yaml backstage-backend-auth-secret
+    apiVersion: v1
+    data:
+      ARGOCD_PASSWORD: ...
+      ARGOCD_URL: ...
+      ARGOCD_USERNAME: ...
+      BACKEND_SECRET: ...
+      GITHUB_TOKEN: ...
+      K8S_CLUSTER_TOKEN: ...
+      K8S_CLUSTER_URL: ...
+    kind: Secret
+    metadata:
+      creationTimestamp: "2024-05-07T22:22:59Z"
+      name: backstage-backend-auth-secret
+      namespace: rhdh-operator
+      resourceVersion: "4402773"
+      uid: 2042e741-346e-4f0e-9d15-1b5492bb9916
+    type: Opaque
+    ```
+1.  Use the following manifest to install the operator in an OCP cluster:
+
+    ```yaml
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: orchestrator-operator
+      namespace: openshift-operators
+    spec:
+      channel: alpha
+      installPlanApproval: Automatic
+      name: orchestrator-operator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+    ```
+
+1.  Run the following commands to determine when the installation is completed:
+
+    ```console
+    wget https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/main/hack/wait_for_operator_installed.sh -O /tmp/wait_for_operator_installed.sh && chmod u+x /tmp/wait_for_operator_installed.sh && /tmp/wait_for_operator_installed.sh
```

Comment:
Should be `https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/release-1.3/hack/wait_for_operator_installed.sh`